sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClientJavaDocCodeSnippets.java

```diff
@@ -355,6 +355,7 @@ public void beginRecognizeReceiptsFromUrlWithOptions() {
         formRecognizerAsyncClient.beginRecognizeReceiptsFromUrl(receiptUrl,
             new RecognizeReceiptsOptions()
                 .setFieldElementsIncluded(includeFieldElements)
+                .setLocale("en-US")
```

Comment:
Is the idea to always set it in the samples?

Reply:
I just wanted to showcase this feature and thought samples would be a good way to do it.